### DISCUSSION TITLE
Receive in-app notification when a contact request fails

### DISCUSSION THREAD

**Discussion by ilmotta** [status-im/status-legacy#16654] [General]
# Feature Issue

### User Story

>As a user
>When I send a contact request that fails
>I want to receive an in-app notification
>So that I know something went wrong

>Given that I have a failed contact request activity
>When I press it
>Then I should be redirected to my profile screen.

### Description

Related to [#13970 Allow user to see failed contact requests](https://github.com/status-im/status-mobile/issues/13970)

This feature is part of a larger effort to create a new Activity Center. Please, reach out to members of the UX team before starting this work in case there are important details not apparent in Figma.

- [Figma - Activity Center user journeys](https://www.figma.com/file/eDfxTa9IoaCMUy5cLTp0ys/Shell-for-Mobile?node-id=639%3A21926)

**Comment by alwx** [comment]
Also related to #13973, will do them all in one go.

**Comment by ilmotta** [comment]
> Also related to #13973, will do them all in one go.

:+1: @alwx, I remember one open topic was *what do we mean by failure*? It wasn't specified by designers in Figma. The issue https://github.com/status-im/status-mobile/issues/13970 lightly mentions this concern.

**Comment by alwx** [comment]
Linked to https://github.com/status-im/status-mobile/pull/14853

I still don't really get what failure is in this context. Who is the best person to ask? (cc @ilmotta)

**Comment by ilmotta** [comment]
> Linked to #14853
> 
> I still don't really get what failure is in this context. Who is the best person to ask? (cc @ilmotta)

@alwx: I think the best course of action here is to talk to designers and nail down the specs, how the app should behave (happy/sad paths), etc. If that's clear then I think the implementation will follow naturally. I just created the issue for future reference, but it's up to you to find out what needs to be done and how.

**Comment by cammellos** [comment]
Moving to discussion until ready to be actioned
